sdk/ai/Azure.AI.Projects/src/Azure.AI.Projects.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is the Azure.AI.Client client library for developing .NET applications with rich experience.</Description>
```

Comment:
Probably want to fill these out with custom values before doing the beta release.